AK/Format.h

```diff
@@ -488,4 +486,4 @@ using AK::CheckedFormatString;
 using AK::FormatIfSupported;
 using AK::FormatString;
 
-#define dbgln_if(flag, fmt, ...) dbgln<flag>(fmt, ##__VA_ARGS__)
+#define dbgln_if(flag, fmt, ...) ((flag) ? dbgln(fmt, ##__VA_ARGS__) : (void)0)
```

Comment:
nit pick: It might be arguably more clear if it was just wrapped in constexpr inside the macro? 

```suggestion
#define dbgln_if(flag, fmt, ...) \ 
  if constexpr (flag) { dbgln(fmt, ##__VA_ARGS__) } 
```

Reply:
This would mess up following `else` clauses, fixing that would turn it to something like `do { if constexpr (...) ... } while(0)`

Either would work, but I like the ternary better :shrug:

Reply:
I guess it could be more clear, why not.